tough/src/editor/signed.rs

```diff
@@ -54,12 +54,9 @@ where
         // Ensure the keys we have available to us will allow us
         // to sign this role. The role's key ids must match up with one of
         // the keys provided.
-        let (signing_key_id, signing_key) = root_keys
+        let valid_keys = root_keys
```

Comment:
If we don't find any keys, I guess we would catch that with the `role.signatures.len()` check below?

Also - if we don't use this error variant anymore, let's make sure it gets deleted.

Reply:
Thats correct, and the error is used later as well.